pad_server/src/state.h

```diff
@@ -11,11 +11,21 @@
 /* Number of actuators in the system: 12 solenoid valves, 1 fire valve, 1 quick disconnect, 1 igniter */
 #define NUM_ACTUATORS (12 + 1 + 1 + 1)
 
+typedef struct {
+    pthread_mutex_t mut;
+    pthread_cond_t cond;
+    enum { ACT, ARM } target;
+    act_id_e act_id;
+    bool act_val;
+    arm_lvl_e arm_lvl;
```

Comment:
You won't need these either; just use the values from `padstate_t` to send. Once you remove these members, you'll need one more member which is a `bool`, like `update_recorded`. This is the predicate to the condvar because there could be spurious wakeups. When you signal the condvar, you set this member to `true`, and when you send the update, you set the member back to `false`.